teuthology/orchestra/remote.py

```diff
@@ -243,8 +259,7 @@ def sh(self, script, **kwargs):
         if 'args' not in kwargs:
             kwargs['args'] = script
         proc=self.run(**kwargs)
-        return proc.stdout.getvalue()
-
+        return proc.stdout.getvalue().strip()
```

Comment:
Please, don't do this here. The value can be multiline, and in general should save formatting.

Reply:
I'm sorry for misleading or misunderstanding.

Reply:
Ok. I'll reverse the change and leave calling `strip()` upto the caller.

Reply:
Done.